cmd/thanos/store.go

```diff
@@ -42,6 +42,9 @@ func registerStore(m map[string]setupFunc, app *kingpin.Application, name string
 	indexCacheSize := cmd.Flag("index-cache-size", "Number of data records that are kept in the index cache.").
 		Default("100000").Int()
 
+	chunkPoolSize := cmd.Flag("chunk-pool-size", "Maximum size of concurrently allocatble bytes for chunks.").
```

Comment:
I am confused with our policy for flag description. Should it punctuated and uppercased? or not? If yes - we need to fix the rest. If not, let's fix here and above (:

I think proper sentences make sense.

Reply:
Yea, I think we overdid the `.` in some places. Let's to a biggish revamp before we do a first release so you don't have to update all manifests now.

Reply:
kk